### PR TITLE
fix(visualizer): Add support for loading time serie

### PR DESCRIPTION
**Pull request by jourdain**

### CODE REVIEW

**Comment by jourdain** [comment]
I've patched ParaView on Ulmus to support the * in the file name using the protocol.py from ParaView/master.

File: server/taskflows/hpccloud/taskflow/pyfr/__init__.py [hunk 1]
```diff
@@ -498,7 +498,21 @@ def upload_output(task, _, cluster, job, *args, **kwargs):
     # step.
     file_names = [f['name'] for f in solution_files]
     file_names.sort()
-    vtu_file = '%s.vtu' % file_names[0].rsplit('.', 1)[0]
+    if len(file_names) > 1:
+        vtu_fileFirst = '%s.vtu' % file_names[0].rsplit('.', 1)[0]
```
Comment (cjh1) [inline server/taskflows/hpccloud/taskflow/pyfr/__init__.py]: can we use snake_case here as in vtu_file_first

File: server/taskflows/hpccloud/taskflow/pyfr/__init__.py [hunk 2]
```diff
@@ -498,7 +498,21 @@ def upload_output(task, _, cluster, job, *args, **kwargs):
...
+        # find where to put the ...*...
+        head = tail = size = len(vtu_fileFirst)
+        for fileName in file_names[1:]:
```
Comment (cjh1) [inline server/taskflows/hpccloud/taskflow/pyfr/__init__.py]: snake_case

Comment (jourdain) [inline server/taskflows/hpccloud/taskflow/pyfr/__init__.py]: sure can, just not use to it. Will push shortly an update

File: server/taskflows/hpccloud/taskflow/pyfr/__init__.py [hunk 3]
```diff
@@ -498,7 +498,21 @@ def upload_output(task, _, cluster, job, *args, **kwargs):
...
+            for i in range(size):
+                if vtu_fileFirst[i] != vtu_name[i] and head > i:
+                    head = i # not included in the cut
+                if vtu_fileFirst[-i] != vtu_name[-i] and tail >= i:
+                    tail = i - 1
+
+        vtu_file = '%s*%s' % (vtu_fileFirst[:head], vtu_fileFirst[-tail:])
+    else:
+        vtu_file = '%s.vtu' % file_names[0].rsplit('.', 1)[0]
     task.taskflow.set_metadata('vtuFile', vtu_file)
```
Comment (cjh1) [inline server/taskflows/hpccloud/taskflow/pyfr/__init__.py]: A general comment this seems pretty complex, can't we just pass *.vtu to paraview?

Comment (jourdain) [inline server/taskflows/hpccloud/taskflow/pyfr/__init__.py]: maybe, but I haven't tried.

Comment (jourdain) [inline server/taskflows/hpccloud/taskflow/pyfr/__init__.py]: Basically playing the time may render various objects depending of the time.

**Comment by jourdain** [comment]
@cjh1 @TristanWright  need a review again so we can merge

**Comment by cjh1** [comment]
Did you try just passing *.vtu to paraview? There will be no other vtu file in this directory.

**Comment by jourdain** [comment]
Unless the user put one in for the context. Like I did for the airplane.

**Comment by cjh1** [comment]
In that case 'prefix*.vtu" would work

**Comment by jourdain** [comment]
Yes but what is the prefix?

**Comment by cjh1** [comment]
The name of the input mesh
```
kwargs['meshFilename'].rsplit('.', 1)[0]

```

**Comment by jourdain** [comment]
I thought it will come from the user in its ini file definition

```
[soln-plugin-writer]
dt-out = 8.451542547285166
basedir = .
basename = taylor_green-{n:03d}
```

**Comment by cjh1** [comment]
True, in the case of a user generated ini file.

**Pull request merged**